composer.json

```diff
@@ -15,11 +15,9 @@
 		"php": ">=7.4.0",
 		"ext-json": "*",
 		"bamarni/composer-bin-plugin": "^1.5",
-		"christophwurst/nextcloud-http-client": "^0.2.0",
```

Comment:
I need to check this. The httplug package *detects* the HTTP client dynamically. I'm curious which implementation it depects without this meta package.

Reply:
I don't remember why this was added. Right now https://packagist.org/packages/web-auth/webauthn-lib doesn't depend on any of the virtual packages provided by https://packagist.org/packages/christophwurst/nextcloud-http-client.

Reply:
I did another test (login, personal settings and setup) and it works fine.